documentation/src/docs/auth/auth-plugin.md

```diff
@@ -0,0 +1,25 @@
+<!--
+Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+# Implement Pravega Authentication/Authorization plugin
+
+This guide describes the authentication/authorization plugin model for Pravega.
```

Comment:
This guide describes in detail the authentication/authorization plugin model for Pravega